common/containers/Tabs/Contracts/components/Interact/components/InteractForm/index.tsx

```diff
@@ -76,6 +77,48 @@ class InteractForm extends Component<Props, State> {
     return contracts && contracts.length;
   };
 
+  public componentDidMount = () => {
+    setTimeout(
+      function() {
+        this.contractSetup();
+        const item = document.getElementById('accessButton');
```

Comment:
It would be nice to have this use the method `AccessContract` instead of relying on something that could change in the future like the ID.

Reply:
Thanks for the great point! I changed it